src/praisonai/praisonai/cli/commands/unified.py

```
@@ -0,0 +1,575 @@
+    console.print()
+    
+    try:
+        uvicorn.run(
+            fastapi_app,
+            host=host,
+            port=port,
+            log_level="info"
+        )
+    except KeyboardInterrupt:
+        console.print("\n[yellow]🌟 Unified Dashboard stopped.[/yellow]")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)